.travis.yml

```diff
@@ -109,6 +108,8 @@ before_script:
   # Restore the original requirements.txt
   - mv requirements.orig requirements.txt
   - python .misc/check_setuptools.py
+  - pip --version
+  - pip install --upgrade pip~=9.0
```

Comment:
this isnt relevant for this issue -- only `check_unsupported.sh` is executed on these old unsupported pythons which have old pip.  In there, the pip needs to be upgraded.

Reply:
@jayvdb I know, but if you check the [previous build](https://travis-ci.org/abhishalya/coala/builds/533653229), it was failing due to pip==9.0 being used. So, I added these two to check.